package.json

```diff
@@ -85,10 +75,10 @@
     "rollup-plugin-multi-entry": "^2.0.2",
     "rollup-plugin-node-resolve": "^3.0.0",
     "rollup-plugin-string": "^2.0.2",
+    "rollup-plugin-uglify": "^2.0.1",
```

Comment:
does this use `uglify-es`?

Reply:
It uses `uglify-js` by default but I can switch to `uglify-es`

Reply:
nah I was just wondering. I think we will want to do some tests to see how stable uglify-es is before we use it.